content/project-news/reactos-in-2020.md

```diff
@@ -0,0 +1,72 @@
+We now have GCC 8.4, CMake 3.17 as our defaults in the new RosBE 2.2, while maintaining compatibility with the latest CMake. The transition didn't happen without flaws though, a lot of effort was put by Colin Finck to make modern CMake work on Windows 2003 (and thus on ReactOS because we maintain compatibility with it). All the patches are freely available in our GitHub.
+
+Clang build is resurrected on the CI, with plans on supporting this compiler along with GCC and MSVC. ReactOS buils with Clang now, but still cannot boot though.
+
+A controversial decision was made to drop support of old MSVC compilers, and raise the C language standard to C99 for better cross-compiler compatibility. The deprecation brought up a big discussion over ML and GitHub.
+Still, this opens a window for future C++ standard library upgrades (maybe in 2021, who knows) and the support of modern C++ in our codebase.
+
+## Kernel and drivers
+
+As there are two developers working full-time in this field, there are quite some long-awaited changes:
+
+**Merge of the PnP storage stack** The new storage stack, derived from open-source Microsoft drivers came to replace the old NT4-compatible ReactOS storage stack. For the long time many workarounds were applied to make NT4 storage stack work in the NT5.2 environment. This era ends now, with a major step forward in compatibility with modern Windows, along with new features:
+* GPT partitioning support
+* Support of the user-mode APIs related to storage stack
+* USB drives mount and dismount on-the-go
+* Special SSD commands, Blu-ray disks, > 2TiB drives and many more
+
+**Kernel-mode Driver Framework support** As a part of the storage stack switch, the KMDF was ported to ReactOS allowing more drivers to potentially be used in the OS. The port is not complete yet, but already allows to use some drivers.
+
+**Memory manager/Common cache subsystems improvements** This is done by Jérôme Gardou, in parallel with the PnP and storage work by Victor Perevertkin. These two subsystems (CC and MM) break records by both the amount of time spent and by the amount of people who worked on them. Current effort is aimed on better compatibility with Windows on running filesystem drivers. And there are quite some achievements here: ReactOS is finally able to work with `ntfs.sys` driver from Windows and can handle an open-source Microsoft `fastfat.sys` driver for FAT filesystem. The work is not finished yet, but is going to be merged in 2021 for sure.
+
+**Plug and Play subsystem improvements** A major refactoring is going on here, to make the driver compatibility more complete and improve the overall system stability. This already made possible for a long-awaited ability to boot from USB media in an off-the-tree branch
+
+All those changes wouldn't be possible without the donations from the community. Big thanks to all of the patrons!
+
+## CMD parser improvements and polishing
+
+_changes by hbelusca_
```

Comment:
```suggestion
_changes by Hermès BÉLUSCA - MAÏTO_
```

Reply:
You used names instead of aliases for other contributors as well ;)

Reply:
oh, you are asking him to fill this in I guess?

Reply:
@Extravert-ir : See my PM in mattermost.